MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -875,6 +875,41 @@
 	_metalFeatures.minSwapchainImageCount = kMVKMinSwapchainImageCount;
 	_metalFeatures.maxSwapchainImageCount = kMVKMaxSwapchainImageCount;
 
+#if MVK_TVOS
+	_metalFeatures.mslVersionEnum = MTLLanguageVersion1_0;
```

Comment:
This should probably be `MTLLanguageVersion1_1`. Metal didn't appear on tvOS until tvOS 9, by which point Metal 1.1 was ready.

Reply:
Done!